### PR TITLE
New languages

### PR DESCRIPTION
**Pull request by RyckRichards**

### CODE REVIEW

File: app/vendors/languages_lib.php [hunk 1]
```diff
@@ -506,7 +507,11 @@ function __($string) {
                 'cos' => __d('languages', 'Corsican',true),
                 'hnj' => __d('languages', 'Hmong Njua',true),
                 'rif' => __d('languages', 'Tarifit',true),
-                
+                'nch' => __d('languages', 'Central Huasteca Nahuatl',true),
+                'kek' => __d('languages', "'Kekchi (Q'eqchi')",true),
```
Comment (jiru) [inline app/vendors/languages_lib.php]: I think there is an extra single quote here.
```php
"'Kekchi (Q'eqchi')"
```
Should be
```php
"Kekchi (Q'eqchi')"
```

**Comment by jiru** [comment]
According to its [Wikipedia article](https://en.wikipedia.org/wiki/Assyrian_Neo-Aramaic), Assyrian is written right to left, so it should be added to the array of right to left languages.

**Comment by RyckRichards** [comment]
Gillux, that was fixed. Thank you so really much!

On 1/11/16, gillux <notifications@github.com> wrote:
> According to its [Wikipedia
> article](https://en.wikipedia.org/wiki/Assyrian_Neo-Aramaic), Assyrian is
> written right to left, so it should be added to the array of right to left
> languages.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/Tatoeba/tatoeba2/pull/977#issuecomment-170676308


-- 

Ricardo Vernaut Jr

Mobile // Celular: (5521) 967-640-668
Email: rvernautjr@gmail.com
<https://www.linkedin.com/in/vernautjr>

File: app/vendors/languages_lib.php [hunk 2]
```diff
@@ -506,7 +507,11 @@ function __($string) {
                 'cos' => __d('languages', 'Corsican',true),
                 'hnj' => __d('languages', 'Hmong Njua',true),
                 'rif' => __d('languages', 'Tarifit',true),
-                
+                'nch' => __d('languages', 'Central Huasteca Nahuatl',true),
+                'kek' => __d('languages', "Kekchi (Q'eqchi')",true),
+                'ssw' => __d('languages', 'Swazi',true),
+                'ban' => __d('languages', 'Balinese',true),
+                'aai' => __d('languages', 'Assyrian',true),
```
Comment (jiru) [inline app/vendors/languages_lib.php]: It’s aii, not aai.

**Comment by jiru** [comment]
I made the necessary changes so that Assyrian characters are taken account by Sphinx and sentences can be found by searching in that language, although it won’t work until we fix #951.

**Pull request merged**